### ISSUE TITLE
[Feature Request] Option for double-click to play

### ISSUE THREAD

**Issue by Dorian-DS** [artemanufrij/playmymusic#179]
Give option in preferences to double-click a playlist item to play it instead of single-click.

Default can remain single-click, but the option to change this would make it more adaptable to users' preferences.

**Comment by maboleth** [comment]
I agree, this is also needed, sometimes I'd like to select a song just for the "next" play, after the current one finishes, shouldn't be immediate.

Not sure, however, how much this player is maintained anymore, devs don't respond this year... :(
